deps/v8/src/mirror-debugger.js

```diff
@@ -1515,7 +1515,7 @@ PropertyMirror.prototype.name = function() {
 
 
 PropertyMirror.prototype.toText = function() {
-  if (IS_SYMBOL(this.name_)) return %SymbolDescriptiveString(this.name_);
+  if (IS_SYMBOL(this.name_)) return %SymbolDescription(this.name_);
```

Comment:
Should the argument be couched in a `%_ValueOf(...)`?  I see other calls to `%SymbolDescription()` do that but I presume that's for the case when `IS_SYMBOL(x) === false && IS_SYMBOL_WRAPPER(x) === true`.